cephbackupstorage/ansible/cephb.py

```
@@ -185,6 +169,29 @@
+if py_version and not py_version.startswith("3.11"):
+    command = "rm -rf %s" % virtenv_path
+    run_remote_command(command, host_post_info)
+    py_version = None
+
+if not py_version:
+    # name: make sure virtualenv has been setup
+    command = "python3.11 -m venv %s --system-site-packages" % virtenv_path
+    run_remote_command(command, host_post_info)
+
+# name: install python pkg and replace ceph python path
+replace_content(ceph_file_path, "regexp='/usr/bin/env python' replace='/usr/bin/python3.11'", host_post_info)
```

Ruff (0.14.7)

`replace_content` may be undefined, or defined from star imports

(F405)